src/saltext/vmware/modules/vmc_org_users.py

```diff
@@ -232,111 +232,238 @@ def invite(
 
-        'expiresAt': (Integer As Int64) (Optional)
-            The timestamp the role expires at (measured in number of seconds since 1/1/1970 UTC). Example:3609941597
+    custom_roles
+        (Optional) List of custom role assigned to a user.
+        It can be specified in the below format
 
-        'lastUpdatedBy': (String) (Optional)
-            Last Updated time of the role.
+        .. code::
 
-        'lastUpdatedDate': (String) (Optional)
-            The timestamp the role was updated at (measured in number of seconds since 1/1/1970 UTC).
+            "custom_roles": [
+                        {
+                            "name": "role_name"
+                        }
+                    ]
+            where 'name' indicates the name of the custom role.
 
-        'membershipType': (String) (Optional)
-            Membership type of the member in the organization.
-            DIRECT: if the member roles were assigned directly.
-            GROUP: if the member roles were assigned through custom or enterprise group.
-            Possible values are: DIRECT , GROUP , NESTED
+    service_roles
+        (Optional) List of service roles to attach to a user.
+        Below fields defines the properties of service roles.
 
-        'createdBy': (String) (Optional)
-            The Creator of the role.
+        'serviceDefinitionLink': (String) (Optional)
+            The link to the service definition.
 
-        'resource': (String) (Optional)
-            The resource in which the role is scoped to. The resource will be embedded in the Access Token
-            "perms" claim, as part of the role.
+        'serviceRoles': list
+            It can be specified in the below format
 
-    skip_notify
-        (Optional) Indicates if the notification have to be skipped.
+            .. code::
 
-    custom_roles
-        (Optional) List of custom role bindings which defines the binding properties of the custom role.
-        Custom role binding is a json object which can contain the below fields.
+                "serviceRoles": [
+                        {
+                            "name": "role_name"
+                        }
+                    ]
+                where 'name' indicates the name of the service role.
 
-        'name': String
-            The name of the role.
+    skip_notify_registration
+        (Optional) Prevent sending mails to users that do not yet have a CSP profile.
 
-        'createdDate': (String) (Optional)
-            The timestamp the role was created at (measured in number of seconds since 1/1/1970 UTC).
+    invited_by
+        (Optional) Indicates the actual user who is inviting.
 
-        'expiresAt': (Integer As Int64) (Optional)
-            The timestamp the role expires at (measured in number of seconds since 1/1/1970 UTC). Example:3609941597
+    custom_groups_ids
+        (Optional) List of unique IDs of Custom Groups. Can have a maximum of 15 custom group IDs.
```

Comment:
**praise** love to see this called out here, that there's a max. That will be super helpful to users.